docs/topics/installing-mta-disconnected-environment.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="installing-mta-disconnected-environment_{context}"]
+= Installing {ProductShortName} on a disconnected environment
+
+On a connected device, first download and save the {ProductShortName} binary. Then download and save the Podman images required, you require the {ProductShortName} Podman image and the required provider image, you require.
+
+* Download the required {ProductShortName} binary from the link:https://developers.redhat.com/products/mta/download[customer portal]:
+** CLI for Linux aarch64
+** CLI for MacOS x86_64
+** CLI for MacOS aarch64
+** CLI for Windows x86_64
+** CLI for Windows aarch64
+
+* On a connected device, download and save the images.
+
+* Copy the binary to the disconnected device.
+
+In addition, you need to save and import the associated container images by using Podman.
+
```

Comment:
```suggestion
* On a connected device, download and save the images.

* Copy the binary to the disconnected device.

* In addition, you need to save and import the associated container images by using Podman.

```

Should the "In addition..." line be a bullet? or sub-point to "Copy the binary"?